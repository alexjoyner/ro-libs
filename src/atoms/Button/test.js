import React from 'react';
import { shallow } from 'enzyme';
import { Button } from './';


describe('Button', () => {
  it('Should render without exploding', () => {
    const component = shallow(<Button />);
    expect(component).toMatchSnapshot();
  });
});
