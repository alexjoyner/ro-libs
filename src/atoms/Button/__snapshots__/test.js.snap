// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component Should match snapshot 1`] = `
<styled.button
  danger={false}
  dark={false}
  ghost={false}
  large={false}
  primary={false}
  small={false}
  success={false}
  type="button"
  warning={false}
  xsmall={false}
>
  <styled.span
    danger={false}
    dark={false}
    ghost={false}
    large={false}
    primary={false}
    small={false}
    success={false}
    tabIndex={-1}
    warning={false}
    xsmall={false}
  >
    Test Button
  </styled.span>
</styled.button>
`;
